rapids/BUILD

```diff
@@ -8,10 +8,13 @@ py_test(
     srcs = ["test_rapids.py"],
     data = [
         "rapids.sh",
+        "//dask:dask.sh",
+        "//dask:verify_dask_standalone.py",
+        "//dask:verify_dask_yarn.py",
```

Comment:
Do we need to run Dask tests in Rapid init action?

Reply:
ack